content/02.introduction.md

```diff
@@ -1,29 +1,25 @@
 ## Introduction
+
+Scientific communication has been around since the beginning of science where the main form of communication was through scientific meetings [@raw:smith2006trouble].

```

Comment:
This leaves out letters sent directly between pairs of scientists. I realize that the Smith paper is asserting that, but is it true? The comment itself in the Smith paper lacks references.

My impression is that communication included direct meetings but also individually addressed letters and books. How did someone like Tycho Brahe communicate?